lib/internal/constants.dart

```diff
@@ -46,7 +61,12 @@ abstract class ArgumentName {
 
   static const String serviceUuid = "serviceUuid";
   static const String serviceId = "serviceId";
```

Comment:
serviceIdentifier?

Reply:
Isn't it even some kind of `internalUniqueServiceIdentifier`?

Reply:
We can consider a better naming for that but for now I will just leave serviceIdentifier for the sake of consistency with similar variables within the lib and MBA